add section: SVG to raster format conversion (#237)
imagekitio/docs-keep-a-changelog: features/image-optimization/svg-to-raster-format-conversion.md

# SVG to raster format conversion

> ImageKit.io allows you to convert SVG images to raster formats by using the [format parameter (f)](../image-transformations/resize-crop-and-other-transformations.md#format---f).

Reviewer: `format parameter` => `format transformation parameter`

> ImageKit.io allows you to convert SVG images to raster formats by using the [format parameter (f)](../image-transformations/resize-crop-and-other-transformations.md#format---f).

Reviewer: ```suggestion
ImageKit.io allows you to convert SVG images to raster formats by using the [format transformation parameter (f)](../image-transformations/resize-crop-and-other-transformations.md#format---f).
```

Author: https://github.com/imagekitio/docs-keep-a-changelog/pull/237#discussion_r1437996069

> To perform this conversion, you need to specify the desired raster format as `f-<raster_format>`. For example: `f-png` or `f-jpg` etc.

Reviewer: ```suggestion
To perform this conversion, you need to specify the desired raster format as `f-<raster_format>` in the image URL. For example: `f-png`, `f-jpg`, etc.
```

Author: https://github.com/imagekitio/docs-keep-a-changelog/pull/237#discussion_r1437996069

{% hint style="info" %}

> If you enable automatic format conversion or use `f-auto`, the SVG images will not be converted to raster formats and will be delivered as SVG files only.

Reviewer: ```suggestion
If you enable automatic format conversion or use `f-auto` in the image URL, SVG images will not be converted to raster formats and will be delivered as SVG files only. Specifying `f-<raster_format>` in the URL is the only way to convert SVGs to raster formats.
```

Reviewer: Also, add a link to the `Automatic format conversion` page

Author: https://github.com/imagekitio/docs-keep-a-changelog/pull/237#discussion_r1437996069

{% endhint %}

> Once the SVG images are converted to raster formats, you can also use other image transformation parameters to resize, filter, and add overlays.

Reviewer: ```suggestion
When converting an SVG image to any raster format, you can also apply other image transformations such as resizing, overlays, etc.
```

Author: https://github.com/imagekitio/docs-keep-a-changelog/pull/237#discussion_r1437996069

{% hint style="info" %}

> SVG to SVG conversion does not support image transformations (such as resizing, adding overlay and applying filters) on SVG images directly.

Reviewer: ```suggestion
To apply image transformations such as resizing, adding overlays, etc., SVG images must be converted to raster formats.
```

Author: https://github.com/imagekitio/docs-keep-a-changelog/pull/237#discussion_r1437996069
